raw_data/preprocessed/africa/zimbabwe_english.txt

```diff
@@ -0,0 +1,486 @@
+
+[Subsection amended by section 28 of Act 6 of 2005]
+
+[3] A person found guilty under subsection [2] shall, in addition to any sentence or fine imposed on him: [a] pay the cost of the removal of the pollution, including any cost which may be incurred by any Government agency in the restoration of the environment damaged or destroyed as a result of the emission, and [b] render to third parties affected by the offence such reparation, restoration, restitution or compensation as may be determined by court upon application by such third parties.
+
+Licensing emissions
+(64) No owner or operator of a trade or any establishment shall emit a substance or energy which causes or is likely to cause air pollution without an emission licence issued by the Board.
+
+Application for emission licence
+(65) [1] An application for an emission licence shall be made to the Board in the prescribed form and be accompanied by the prescribed fee. [2] Before issuing a licence in respect of emissions, the Board shall: [a] consider the possible effects of the emissions on the quality of ambient air; [b] consider existing licences affecting the same air resource; [c] give due regard to the requirements of residents, human settlements and other industrial and commercial activities. [3] An emission licence issued under this Act shall be subject to such conditions as may be prescribed or as may be specified in the licence and shall remain valid for such period and may be renewed for such further period as may be prescribed or specified in the licence. [4] Where the Board rejects an application for the grant of an emission licence, it shall within twenty-one days of its decision, notify the applicant in writing of its reasons for such refusal.
+
+Cancellation of emission licence
+(66) The Board may, in writing, cancel any emission licence [a] if the holder of the licence contravenes any provision of this Act or regulations made thereunder; [b] if the holder fails to comply with any conditions specified in the licence; or [c] if the Board considers it in the interest of the environment or in the public interest to so to do.
+
+Register of emission licences
+(67) [1] The Board shall maintain a register of all emission licences issued under this Act. [2] The register shall be a public document and may be inspected at a reasonable hours by any person on the payment of a prescribed fee.
```

Comment:
In the version of the legislation from the spreadsheet, the sentences in this article are not numbered (i.e. there is no `[1]` or `[2]`).